pkg/netceptor/conn_tls_test.go

```diff
@@ -0,0 +1,125 @@
+package netceptor_test
```

Comment:
these tests can be moved into the netceptor package, perhaps then you would need to expose so many functions